docs/install-upgrade/config.md

```diff
@@ -208,10 +90,89 @@ spec:
   bootstrap:
     disk: "/dev/sda" # disk to install OS on, e.g. "sda" or "nvme0n1"
   external:
-    interface: eno2 # interface for external
+    interface: eno2 # customer interface to manage control node
     ip: dhcp # IP address for external interface
-  management:
+  management: # interface that manages switches in private management network
     interface: eno1
 
 # Currently only one ControlNode is supported
 ```
+
+### Configure Control Node and Switch Users
+
+#### Control Node Users
+Configuring control node and switch users is done either passing 
+`--default-password-hash` to `hhfab init` or editing the resulting `fab.yaml` 
+file emitted by `hhfab init`.  The default username on the control node is
+core. 
+
+#### Switch Users
+There are two users on the switches, `admin` and `operator`. The `operator` user has
+read-only access to `sonic-cli` command on the switches. The `admin` user has
+broad administrative power on the switch. 
+In order to avoid conflicts, do not use the following usernames: `operator`,`hhagent`,`netops`.
+
+### NTP and DHCP
+The control node uses public ntp servers from cloudflare and google by default.
+The control node runs a dhcp server on the management network. See the [example file](#complete-example-file).
+
+### Control Node
+The control node is the host that manages all the switches, runs k3s, and serves images. 
+The **management** interface is for the control node to manage the fabric 
+switches, *not* end-user management of the control node. For end-user 
+management of the control node specify the **external** interface name.
+
+### Telemetry
+
+There is an option to enable [Grafana
+Alloy](https://grafana.com/docs/alloy/latest/) on all switches to forward metrics and logs to the configured targets using
+[Prometheus Remote-Write
+API](https://prometheus.io/docs/specs/prw/remote_write_spec/) and Loki API. Metrics includes port speeds, counters, 
+errors, operational status, transceivers, fans, power supplies, temperature
+sensors, BGP neighbors, LLDP neighbors, and more. Logs include Hedgehog agent logs.
+
+Telemetry can be enabled after installation of the fabric. Open the following
+YAML file in an editor on the control node. Modify the fields as needed. Logs
+can be pushed to a grafana instance at the customer environment, or to grafana
+cloud. To enable the telemetry after install use:
+
+``` shell
+kubectl patch -n fab --type merge fabricator/default --patch-file telemetry.yaml
+```
```

Comment:
> Open the following YAML file [...]
> <Here's a shell command to do something else>
> <Here's a YAML file, oh right that's the one we mentioned at some point>

-> I'd move the shell command and related sentence below the YAML file.

Reply:
Done

Reply:
Hey but I didn't mean to move the whole paragraph!